posthog/hogql_queries/insights/trends/trends_query_runner.py

```diff
@@ -118,14 +121,27 @@ def to_queries(self) -> List[ast.SelectQuery | ast.SelectUnionQuery]:
                 else:
                     query_date_range = self.query_previous_date_range
 
-                query_builder = TrendsQueryBuilder(
-                    trends_query=series.overriden_query or self.query,
-                    team=self.team,
-                    query_date_range=query_date_range,
-                    series=series.series,
-                    timings=self.timings,
-                    modifiers=self.modifiers,
-                )
+                query_builder: TrendsQueryBuilder | DataWarehouseTrendsQueryBuilder
```

Comment:
Very much a lil nit, but, this could be typed as `TrendsQueryBuilderAbstract` now I think